docs/introduction/PriorArt.md

```diff
@@ -9,7 +9,7 @@ Can Redux be considered a [Flux](https://facebook.github.io/flux/) implementatio
 
 (Don’t worry, [Flux creators](https://twitter.com/jingc/status/616608251463909376) [approve of it](https://twitter.com/fisherwebdev/status/616286955693682688), if that’s all you wanted to know.)
 
-Redux was inspired by several important qualities of Flux. Like Flux, Redux prescribes that you concentrate your model update logic in a certain layer of your application (“stores” in Flux, “reducers” in Redux). Instead of letting the application code directly mutate the data, both tell you to describe every mutation as a plain object called an “action”.
+Redux was inspired by several important qualities of Flux. Like Flux, Redux prescribes that you concentrate your model update logic in a certain layer of your application (“stores” in Flux, “reducers” in Redux). Instead of letting the application code directly mutate the data, both let you to describe every mutation as a plain object called an “action”.
```

Comment:
"both let you to" is not grammatically correct. I think "tell" is acceptable here as it's describing a system that prescribes/requires this patterns instead of just allowing it.